### ISSUE TITLE
ERROR...Unsupported query type: DIVAProtocolPolygon

### ISSUE THREAD

**Issue by spuddyminer** [tellor-io/disputable-values-monitor#66]
The disputable-values-monitor errored out with the following traceback:

```
ERROR   | telliot_feeds.queries.abi_query | Unsupported query type: DIVAProtocolPolygon
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/ubuntu/.cache/pypoetry/virtualenvs/tellor-disputables-22F-CnI2-py3.9/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/ubuntu/.cache/pypoetry/virtualenvs/tellor-disputables-22F-CnI2-py3.9/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/ubuntu/.cache/pypoetry/virtualenvs/tellor-disputables-22F-CnI2-py3.9/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ubuntu/.cache/pypoetry/virtualenvs/tellor-disputables-22F-CnI2-py3.9/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/ubuntu/.cache/pypoetry/virtualenvs/tellor-disputables-22F-CnI2-py3.9/lib/python3.9/site-packages/telliot_core/cli/utils.py", line 18, in wrapper
    return asyncio.run(f(*args, **kwargs))
  File "/usr/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/home/ubuntu/disputable-values-monitor/src/tellor_disputables/cli.py", line 41, in main
    await start(all_values=all_values, wait=wait)
  File "/home/ubuntu/disputable-values-monitor/src/tellor_disputables/cli.py", line 84, in start
    new_report = await parse_new_report_event(event, eth_web3, eth_contract)
  File "/home/ubuntu/disputable-values-monitor/src/tellor_disputables/data.py", line 224, in parse_new_report_event
    if receipt["event"] != "NewReport":
TypeError: 'NoneType' object is not subscriptable
```

**Comment by spuddyminer** [comment]
I think this is an old error from an outdated version. Closing now.

**Comment by spuddyminer** [comment]
opened back up because I'm not 100% sure

**Comment by tallywiesenberg** [comment]
It's true, this is uncaught. A fix is coming in #69
